sdcm/tester.py

```diff
@@ -243,6 +243,18 @@ def setUp(self):
             self.log.info('repair system_auth keyspace ...')
             node.run_nodetool(sub_cmd="repair", args="-- system_auth")
 
+        alternator_port = self.params.get('alternator_port', default=None)
+        if alternator_port and not cluster.Setup.REUSE_CLUSTER:
+            # TODO: remove once
+            # since https://github.com/scylladb/scylla-enterprise/issues/1003
+            self.log.info('change RF of alternator keyspace to %s' % system_auth_rf)
+            node = self.db_cluster.nodes[0]
+            with self.cql_connection_patient(node) as session:
+                session.execute(
+                    "ALTER KEYSPACE alternator WITH replication = {'class': 'org.apache.cassandra.locator.SimpleStrategy', 'replication_factor': 3} AND durable_writes = true;")
+            self.log.info('repair alternator keyspace ...')
+            node.run_nodetool(sub_cmd="repair", args="-- alternator")
+
```

Comment:
The issue referred here is already solved so we need to remove this functionality.
10x

Reply:
I see it is gone now :)
Thanks,